Ollivanders/src/net/pottercraft/ollivanders2/spell/APPARATE.java

```diff
@@ -295,6 +302,31 @@ private boolean canApparateTo(@NotNull Location destination)
             return false;
         }
 
+        // check if x or z coordinates exceed max set in configuration
```

Comment:
Usually we load all config globally so we dont have to read the file every time we take an action. This will improve performance since file reads are a lot slower than in-memory variables.